src/main/java/net/pms/configuration/FormatConfiguration.java

```diff
@@ -233,12 +246,14 @@ public boolean match(String container, String videoCodec, String audioCodec) {
 		 * or 0, its value is skipped for making the match. If any of the
 		 * non-null parameters does not match, false is returned. For example,
 		 * assume a configuration that contains only the following line:
-		 *
-		 * Supported = f:mp4 n:2
-		 *
-		 * match("mp4", null, null, 2, 0, 0, 0, 0, null) = true
-		 * match("mp4", null, null, 6, 0, 0, 0, 0, null) = false
-		 * match("wav", null, null, 2, 0, 0, 0, 0, null) = false
+		 * 
+		 * Supported = f:mp4 n:2 se:SUBRIP
+		 *  
+		 * match("mp4", null, null, 2, 0, 0, 0, 0, null, null) = true
+		 * match("mp4", null, null, 6, 0, 0, 0, 0, null, null) = false 
+		 * match("wav", null, null, 2, 0, 0, 0, 0, null, null) = false
+		 * match("mp4", null, null, 2, 0, 0, 0, 0, null, "SUBRIP") = true
+		 * match("mp4", null, null, 2, 0, 0, 0, 0, null, "sub") = false 
```

Comment:
MICRODVD instead of "sub" ?

And what about adding comment in the _DefaultRenderer.conf_ file ?

I'll test it this week.